sidechain/consensus/aura/src/lib.rs

```diff
@@ -104,7 +104,7 @@ impl<AuthorityPair, ParentchainBlock, SidechainBlock, Environment, OcallApi, Imp
 
 /// The fraction of total block time we are allowed to be producing the block. So that we have
 /// enough time send create and send the block to fellow validateers.
-pub const BLOCK_PROPOSAL_SLOT_PORTION: f32 = 0.4;
+pub const BLOCK_PROPOSAL_SLOT_PORTION: f32 = 0.7;
```

Comment:
The commit message says
>Revert some of the investigation changes (log levels for example)
>Also increased the slot fraction for calls again to 0.7 (from 0.4)

But the original value was `0.8`, typo maybe?

Reply:
Just seeing your comment regarding this in https://github.com/integritee-network/worker/pull/1087#discussion_r1011539150
It is now clear to me why it was reduced to `0.7`.